client/src/components/question/ThirdPartyIcons.js

```diff
@@ -80,11 +83,13 @@ class ThirdPartyIcons extends React.Component {
 
     let query;
     if (question.type === "tossup") {
-      query = isPresent(question.formatted_answer) ?
-        question.formatted_answer : question.answer;
-    } else {
-      query = isPresent(question.formatted_answers[index]) ?
-        question.formatted_answers[index] : question.answers[index];
+      query = isPresent(question.formatted_answer)
+		? question.formatted_answer
+		: question.answer;
```

Comment:
I don't believe I have a linter installed is this just your personal preference, or did you have Prettier (e.g.) running on your workspace when you were modifying?

Reply:
Personal preference.